### PR TITLE
Added code to trigger Primary Button on enter key press

### PR DESCRIPTION
**Pull request by PriyankaGitHub7**
# Pull Request Title

Added handler to trigger button clicks on enter key press. 

### Notes
Any notes that need to have attention drawn to them

@darwinfroese
I have added the enter press handlers for Login, Create Account and Create Project pages. Please let me know if there are any other screens to be modified for this.

### CODE REVIEW

File: webclient/src/components/views/Login.vue [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <div>
     <LoginHeader />
     <!-- TODO: All of these inputs should accept 'enter' as the default action -->
-    <div class='container'>
+    <div class='container' v-on:keyup.enter = "enterKeyPressHandler">
```
Comment (darwinfroese) [inline webclient/src/components/views/Login.vue]: This one isn't working for me. Did you mean to put it on `card` like the rest?

**Comment by PriyankaGitHub7** [comment]
Sure, will make the changes on both the pull requests by today or tomorrow.
Yes, my intention was to put it on the card div.

**Comment by PriyankaGitHub7** [comment]
@darwinfroese Resolved the merge conflict and made the requested changes.

**Comment by darwinfroese** [comment]
Looks good!

**Pull request merged**